webapp/scripts/prepareEe.js

```diff
@@ -17,9 +17,10 @@ const eeModuleLocal = path.join(
   '../src/eeSetup/eeModule.current.tsx'
 );
 
-if (existsSync(eeModuleLocal)) {
+try {
+  lstatSync(eeModuleLocal);
   unlinkSync(eeModuleLocal);
-}
+} catch {}
```

Comment:
Hi Aditya, this is a great catch. Thank you. 🙇

It took me some time to understand the bug and I would not get it without your PR description. Would it be OK with you to make the fix easier to understand even to the slower or us? 🦥

I think either

```suggestion
try {
  unlinkSync(eeModuleLocal);
} catch {}
```

or

```suggestion
if (!!lstatSync(eeModuleLocal, { throwIfNoEntry: false })) {
  unlinkSync(eeModuleLocal);
}
```

could be easier to understand.

What is your opinion please?

Reply:
Makes sense.
Thanks for point that out Jan!